### PR TITLE
Passes around lists of project ids rather than project names

### PR DESCRIPTION
**Pull request by AmandaBirmingham**

### CODE REVIEW

**Comment by AmandaBirmingham** [comment]
@wasade @dhakim87 Can you stomach a -0.06% change in test coverage?  If so, the rest of this build passes ... :)

**Comment by wasade** [comment]
Not a problem :) I'm going to defer merge though until a release is demarked which I'm working on right now

File: microsetta_private_api/repo/admin_repo.py [hunk 1]
```diff
@@ -653,24 +653,30 @@ def create_kits(self, number_of_kits, number_of_samples, kit_prefix,
...
-                if name not in known_projects:
-                    raise KeyError("%s does not exist" % name)
-                if known_projects[name][1]:
+            for input_proj_id in project_ids:
+                if input_proj_id not in known_projects:
+                    raise KeyError("Project id %s does not exist" %
+                                   input_proj_id)
+                # if *any* of the projects the kits will be associate with are
+                # microsetta projects, set is_tmi to true
+                curr_tmi = bool(known_projects[input_proj_id])
```
Comment (wasade) [inline microsetta_private_api/repo/admin_repo.py:678]: Would it be possible to do the cast to `bool` within the dict comprehension on line 670?

**Review by wasade — CHANGES_REQUESTED** [review]
Just a small request, thanks!!

**Pull request merged**